packages/block-library/src/gallery/edit.js

```diff
@@ -534,6 +536,7 @@ function GalleryEdit( props ) {
 				/>
 			</BlockControls>
 			{ noticeUI }
+			<style>{ style }</style>
```

Comment:
The additional style tag added inline can cause issues (block gap with previous block ...), ideally it should use the same mechanism used for layout styles to inject the style tags (I think it's a Head slot or something like that)

Reply:
Thanks  - have switched it out for a similar approach to the layout styles as suggested.

Reply:
@youknowriad I have moved [this to a portal ](https://github.com/WordPress/gutenberg/pull/38164/files#diff-d4b42154fbbf15c5420e71e96e5981059c2a9727112b396792be96168cbe0221R18)as used by the layout lib. Do you have any other feedback on how the block gap is now being used in this PR?